include/gz/sensors/AirFlowSensor.hh

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef GZ_SENSORS_AIRFLOWSENSOR_HH_
+#define GZ_SENSORS_AIRFLOWSENSOR_HH_
+
+#include <memory>
+
+#include <sdf/sdf.hh>
+
+#include <gz/utils/SuppressWarning.hh>
+
+#include <gz/sensors/config.hh>
+#include <gz/sensors/air_flow/Export.hh>
+
+#include "gz/sensors/Sensor.hh"
+
+namespace gz
+{
+  namespace sensors
+  {
+    // Inline bracket to help doxygen filtering.
+    inline namespace GZ_SENSORS_VERSION_NAMESPACE {
+    //
+    /// \brief forward declarations
+    class AirFlowSensorPrivate;
+
+    /// \brief AirFlow Sensor Class
+    ///
+    /// A sensor that reports air speed through differential pressure readings.
```

Comment:
Can you update this comment and add a good description of how this sensor works?